crates/stages/stages/src/stages/headers.rs

```diff
@@ -136,7 +135,7 @@ where
                     .map_err(|err| StageError::Fatal(Box::new(err)))?
                     .into();
 
-            let (header, header_hash) = sealed_header.split();
+            let (header, header_hash) = sealed_header.clone().split();
```

Comment:
we can avoid this clone here
```suggestion
            let (header, header_hash) = sealed_header.sealed_ref().split();
```